components/app.js

```diff
@@ -28,7 +36,37 @@ class App{
             error: this.handleGetGradesError
         })
     }
+
     start(){
         this.getGrades();
+        this.gradeForm.onSubmit(this.createGrade);
+    }
+
+    createGrade(name, course, grade){
+        $.ajax({
+            method: "POST",
+            url: "https://sgt.lfzprototypes.com/api/grades",
+            data:
+            {
+                "name": name,
+                "course": course,
+                "grade": grade
+            },
+            headers: { "X-Access-Token": "4zYBwVu7" },
+            success: () => {
```

Comment:
Rather than using an anonymous function with a function call within it here, you should use a callback function 👍

Reply:
For this, I originally had it as a callback but It was referencing the POST request as 'this'. 

![error](https://user-images.githubusercontent.com/61341083/81253289-13be3b80-8fdd-11ea-8d9b-fb461b0f64b0.png)